nitro-protocol/contracts/interfaces/IMultiAssetHolder.sol

```diff
@@ -69,13 +69,20 @@ interface IMultiAssetHolder {
     function reclaim(ReclaimArgs memory reclaimArgs) external;
 
     /**
-     * @dev Indicates that a deposit has been made.
+     * @dev Indicates that an ERC20 deposit has been made.
      * @param destination The channel being deposited into.
-     * @param asset The asset being deposited. Zero address indicates the native token (e.g. ETH).
+     * @param asset The asset being deposited.
      * @param destinationHoldings The new holdings for `destination`.
      */
     event Deposited(bytes32 indexed destination, address asset, uint256 destinationHoldings);
 
+    /**
+     * @dev Indicates that an ETH deposit has been made.
+     * @param destination The channel being deposited into.
+     * @param destinationHoldings The new holdings for `destination`.
+     */
+    event DepositedEth(bytes32 indexed destination, uint256 destinationHoldings);
```

Comment:
If we didn't use a new slimmer `DepositedEth` event but instead used `Deposited` (with a zero asset) when we deposited eth I wonder what the gas difference would be.

Reply:
I had done this in stages, and the slimmer emit is a similar cost saving to the leaner function args list from `deposit_eth`.

(rummaging around for numbers...)

With `deposit_eth` emitting the existing event, the gas reduction for eth deposits was 303. When emitting the slimmer event, the reduction is 571.

Reply:
> Leaving the existing `deposit` function as is and use the existing `Deposited` event would mean the downstream changes from this are pretty minimal. All our existing code would continue to work but we would have the option of optimizing our code by calling `depositEth` when depositing eth.

This is a great point. Will undo the alteration to `deposit`.